fashion-example-query/fashion/app.py

```diff
@@ -114,23 +113,25 @@ def download_data(target: dict, download_proxy=None):
                 v['data'] = load_mnist(v['filename'])
 
 
-def index_generator(num_doc, target: dict):
+def index_generator(num_doc: int, target: dict):
     for j in range(num_doc):
+        d = Document(content=target['index']['data'][j])
```

Comment:
I think the pattern is to do

```
with Document(...) as d:
   d.content = 
```

and then the `update_id` is done by the context managing,no?

Reply:
Since we were talking about maybe not having this context manager anymore, I thought to keep it like this.